### PR TITLE
GitHub Actions for full Nextclade runs

### PR DESCRIPTION
**Pull request by ivan-aksamentov**
### Description of proposed changes    

This PR contains new GitHub Actions workflows from https://github.com/nextstrain/ncov-ingest/pull/214.

I want to experiment with different options for full Nextclade runs (the experiments are in the #215, branch `nextclade-full-run-2021-09-30`, which combines #214 + some more stuff). The `.github/workflows/*` files need to be on master in order for GitHub actions to be triggerable. 

The triggers for the new workflows are currently manual.

Note that the new workflows temporarily hardcode the branch of #215 as follows:

```
    - uses: actions/checkout@v2
      with:
        ref: 'nextclade-full-run-2021-09-30'
```

We need the code from the branch to be able to trigger full runs. Once the code from #214 is on master, the hardcoded branch can (and need to) be removed.

### Related issue(s)  

 - #214 
 - #215

### Testing

Basically the [#215](#215) is my testing playground.

### CODE REVIEW

**Comment by ivan-aksamentov** [comment]
I realized that we don't need to mix the usual ingest flow with Nextclade full run. It can be made separate. The new version is here: https://github.com/nextstrain/ncov-ingest/pull/218
